### DISCUSSION TITLE
Standardize case for Artist names from different sources

### DISCUSSION THREAD

**Discussion by bigoulours** [beetbox/beets#4773] [Ideas]
Sometimes imported albums are not found in Musicbrainz but through other sources (Discogs, Beatport.. ).
However, it happens that some characters in the artist name differ in their case (e.g.: Agent of Time vs Agent Of Time).
Would it make sense to add an option to keep Musicbrainz name as soon as there is a case insensitive match?

**Comment by sampsyo** [comment]
We don't have a way to automate this, really, although it's an interesting idea. Maybe it would be a fun activity to prototype a simple plugin that just tries to canonicalize artists who only differ on case?
